### PR TITLE
fix: [PL-15052]: no data on accounts search

### PR DESCRIPTION
**Pull request by swarajcheguri-harness**
##### Summary:
image

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/10-common/modals/SwitchAccount/SwitchAccount.tsx [hunk 1]
```diff
@@ -216,9 +216,16 @@ const SwitchAccount: React.FC<SwitchAccountProps> = ({ searchString = '', mock }
         {error ? (
           <PageError message={error.message || getString('somethingWentWrong')} onClick={() => refetch()} />
         ) : null}
-        {!loading && !settingDefault && !error && accounts ? (
+        {!loading && !settingDefault && !error && accounts && accounts.length ? (
           <TableV2 columns={columns} data={accounts} sortable={false} />
         ) : null}
+        {!loading && !settingDefault && !error && accounts.length === 0 ? (
```
Comment (meenakshi-harness) [inline src/modules/10-common/modals/SwitchAccount/SwitchAccount.tsx:222]: Both conditions are same expect accounts.length. Can we render conditionally on` accounts.length` by nesting conditional operator?

File: src/modules/10-common/modals/SwitchAccount/SwitchAccount.tsx [hunk 2]
```diff
@@ -216,9 +216,16 @@ const SwitchAccount: React.FC<SwitchAccountProps> = ({ searchString = '', mock }
...
+          <NoDataCard
+            message={getString('noData')}
+            className={css.noDataCard}
+            containerClassName={css.noDataCardContainer}
+          />
+        ) : null}
```
Comment (meenakshi-harness) [inline src/modules/10-common/modals/SwitchAccount/SwitchAccount.tsx:228]: Also should we have svg experience for No Data .. just like we have for other entities?

Comment (swarajcheguri-harness) [reply src/modules/10-common/modals/SwitchAccount/SwitchAccount.tsx:228]: yes ideally we should, but i don't have designs for this now

**Review by meenakshi-harness — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

File: src/modules/75-ce/components/COGatewayConfig/__tests__/COHealthCheckTable.test.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-disabled-tests */
```
Comment (meenakshi-harness) [inline src/modules/75-ce/components/COGatewayConfig/__tests__/COHealthCheckTable.test.tsx:1]: remove

**Review by meenakshi-harness — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Pull request merged**